stories/module-pagespeed-insights-components.stories.js

```diff
@@ -122,6 +122,51 @@ storiesOf( 'PageSpeed Insights Module/Components', module )
 			</WithTestRegistry>
 		);
 	} )
+	.add( 'Dashboard widget (No Recommendations)', () => {
+		const url = fixtures.pagespeedMobile.loadingExperience.id;
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( MODULES_PAGESPEED_INSIGHTS ).receiveGetReport(
+				fixtures.pagespeedMobileNoStackPacks,
+				{
+					url,
+					strategy: STRATEGY_MOBILE,
+				}
+			);
+			dispatch( MODULES_PAGESPEED_INSIGHTS ).finishResolution(
+				'getReport',
+				[ url, STRATEGY_MOBILE ]
+			);
+
+			dispatch( MODULES_PAGESPEED_INSIGHTS ).receiveGetReport(
+				fixtures.pagespeedDesktopNoStackPacks,
+				{
+					url,
+					strategy: STRATEGY_DESKTOP,
+				}
+			);
+			dispatch( MODULES_PAGESPEED_INSIGHTS ).finishResolution(
+				'getReport',
+				[ url, STRATEGY_DESKTOP ]
+			);
+
+			dispatch( CORE_SITE ).receiveSiteInfo( {
+				referenceSiteURL: url,
+				currentEntityURL: null,
+			} );
```

Comment:
Please, use the `provideSiteInfo` helper instead of calling the `receiveSiteInfo` action directly.

Reply:
I've updated this and the other stories in the file to use `provide.*` utility helpers where possible :+1: